lib/topics.js

```diff
@@ -8,18 +8,37 @@ const SNS    = require('./sns');
 class Topic {
 
   constructor(server, topicName) {
-    this._server = server;
-    this.name    = topicName;
+    this._server     = server;
+    this.name        = topicName;
+    this.subscribers = [];
```

Comment:
Note: array vs Set have different semantics. Array allows you to register the same function multiple times, receive the same notification multiple times, but without distinction. Say, you have five users and you want to register a subscriber for each one. One notification for all five would work, but five calls to the same function won't, unless you can distinguish, either by including a context with the notification (i.e `subscribe({ user: 1 }, handler)`) or by using different functions (closures). A set could force you into the correct behavior.